_data/navigation.yml

```diff
@@ -134,10 +134,12 @@ docs:
           - title: "Styling and Page Layout ➢"
             url: /styling-and-page-layout/
             subchildren:
-                - title: "Global Variables"
-                  url: /global-variables/
                 - title: "CSS Architecture"
                   url: /css-architecture/
+                - title: "Directionality"
```

Comment:
I assume this is about an order of the sub pages. If so, I prefer to have it after the page layout docs.

Reply:
Fixed 🙂